lib/cdo/web_purify.rb

```diff
@@ -25,7 +25,7 @@ def self.find_potential_profanities(text, language_codes = ['en'])
     url = "http://api1.webpurify.com/services/rest/" \
       "?api_key=#{CDO.webpurify_key}" \
       "&method=webpurify.live.return" \
-      "&text=#{URI.encode(text)}" \
+      "&text=#{CGI.escape(text)}" \
```

Comment:
I was running into issues with  `URI.encode` not correctly encoding `&` and came across a bunch of stuff online about the issue and that URI.encode / URI.escape is being deprecated and to use CGI.escape instead. It fixed my issue, and another issue @Erin007 and I were seeing with URI.encode breaking on non-ascii characters.

Reply:
nice find!